Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1910,6 +1912,35 @@ def get_sensor_id_command(client: Client, args: dict):
         return CommandResults(readable_output=f"Available Sensor IDs are {output}")
 
 
+def fetch_machine_details_command(client: Client, args: dict):
+    machine_name = str(args.get('machineName'))
+    json_body = {
```

Comment:
Hey, what do you think about adding an argument of limit? 
what happened if the user have more then 1000 results?

Reply:
My apologies for missing it.

Reply:
Hi @sapirshuker 
Sure, we shall include the limit argument as in very extreme cases there might be a possibility of the user having 1000 machines with same name

Reply:
Hey, @suraj-metron 
attached the best practice for [pagination in integration commands](  https://xsoar.pan.dev/docs/integrations/code-conventions#pagination-in-integration-commands )

Reply:
Hi @sapirshuker 
Thank you for the guide. I shall follow it and will make the necessary changes